src/components/Navbar.css

```diff
@@ -0,0 +1,72 @@
+.navbar {
```

Comment:
Ack if we're going to have custom css like this then maybe it's worth adding in Tailwind. If you do decide to use this CSS instead of a Material UI component or something lmk and we'll merge in the Tailwind PR. 

I personally think the dev velocity from Tailwind is worth it, but also I've worked with it before. Let me know if you think the learning curve isn't worth it? I don't have to dictate what we do lol

Reply:
Yes yes let's work with Tailwind